### DISCUSSION TITLE
Use for custom boards with empty EPROMs 27C4001 (4Mbit), 27C801 (8Mbit), 27C160 (16Mbit), 27C322 (32Mbit)

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#113] [Ideas & Suggestions]
Hi Sanni,

I have my own boards with different EPROM size like : 
27C4001 (4Mbit), 27C801 (8Mbit), 27C160 (16Mbit), 27C322 (32Mbit).
Is it possible to use your CartReader and my boards with my already soldered blank EPROM on it ?

**Comment by sanni** [comment]
No. The only eprom supported is the 27C322 and it needs to be in the special eprom adapter to supply the higher voltages needed and even then it might not work since I have stopped working on the eprom code. But it did work once with the one 27C322 eprom I had.

**Reply by StanMOI** [reply]
Thank you for answer
